cpu/stm32f4/include/cpu_conf.h

```diff
@@ -50,7 +50,11 @@ extern "C" {
  * @{
  */
 #define CPU_DEFAULT_IRQ_PRIO            (1U)
+#if defined(CPU_MODEL_STM32F413ZH) || defined(CPU_MODEL_STM32F413VG)
+#define CPU_IRQ_NUMOF                   (102U)
```

Comment:
You state here, that the `stm32f413xx` CPUs have 102 interrupt lines, but the vector definitions in the `vectors.c` file don`t reflect this. So it seems to me, that the `vectors.c` file need adaption as well?